README.md

```diff
@@ -23,7 +23,7 @@ Create a `Clock` component updating the time every second.
 1. Change the `Clock` component to print the time not only on the page but also in the `DevTools` console.
```

Comment:
You didn't realised this task.
Also some css required.

Reply:
done!

Reply:
thank you for help!